src/view/domconverter.js

```diff
@@ -651,4 +654,29 @@ export default class DomConverter {
 	isDocumentFragment( node ) {
 		return node && node.nodeType == Node.DOCUMENT_FRAGMENT_NODE;
 	}
+
+	/**
+	 * Returns `true` if given selection is a backward selection, that is, if it's `focus` is before `anchor`.
+	 *
+	 * @param {Selection} DOM Selection instance to check.
+	 * @return {Boolean}
+	 */
+	isDomSelectionBackward( selection ) {
```

Comment:
Hmm, I thought that I commented on it, but there are multpile methods in `DomConverter` that actually should be moved to `uitls/dom`. This included.